src/ensembl/src/shared/privacy-banner/privacy-banner-service.ts

```diff
@@ -0,0 +1,33 @@
+import storageService, {
+  StorageServiceInterface
+} from 'src/services/storage-service';
+
+export const PrivacyConfig = {
```

Comment:
I am wondering if we should have the privacy config in its own file rather than inside the privacy banner service 🤔

Reply:
I would say, in its own file inside the privacy-banner folder.

Reply:
shall I go ahead and export the configs in another file? Is that what you meant?

Reply:
Yes, go ahead.